src/aspire/basis/fspca.py

```diff
@@ -80,6 +80,11 @@ def __init__(
             len(self.complex_indices_map) == self.complex_count
         ), f"{len(self.complex_indices_map)} != {self.complex_count}"
 
+        # Map Cached SteerableBasis2D index maps
+        self.zero_angular_inds = self.basis.zero_angular_inds
+        self.pos_angular_inds = self.basis.pos_angular_inds
+        self.neg_angular_inds = self.basis.neg_angular_inds
+
```

Comment:
private vars again? (see comment in `steerable.py`)

Reply:
Yep. Changed.